plugin/advisor/advisor.go

```diff
@@ -202,16 +201,16 @@ func Check(dbType db.Type, advType Type, ctx Context, statement string) ([]Advic
 }
 
 // IsSyntaxCheckSupported checks the engine type if syntax check supports it.
-func IsSyntaxCheckSupported(engine db.Type) bool {
-	if engine == db.MySQL || engine == db.TiDB {
+func IsSyntaxCheckSupported(dbType DBType) bool {
+	if dbType == MySQL || dbType == TiDB || dbType == Postgres {
```

Comment:
We disabled the PG syntax check because the current PG parser can not work fine.

See https://github.com/bytebase/bytebase/pull/1656

Reply:
Fixed now